src/webview/add-service-binding/app/addServiceBindingForm.tsx

```diff
@@ -0,0 +1,201 @@
+    const [selectedServiceTouched, setSelectedServiceTouched] = React.useState<boolean>(false);
+    const [bindingNameTouched, setBindingNameTouched] = React.useState<boolean>(false);
+
+    const createVscodeTheme = (paletteMode: PaletteMode): Theme => {
+        const computedStyle = window.getComputedStyle(document.body);
+        return createTheme({
+            palette: {
+                mode: paletteMode,
+                primary: {
+                    main: computedStyle.getPropertyValue('--vscode-button-background'),
+                },
+                error: {
+                    main: computedStyle.getPropertyValue('--vscode-inputValidation-errorBorder'),
+                },
+                warning: {
+                    main: computedStyle.getPropertyValue('--vscode-inputValidation-warningBorder'),
+                },
+                info: {
+                    main: computedStyle.getPropertyValue('--vscode-inputValidation-infoBorder'),
+                },
+                success: {
+                    main: computedStyle.getPropertyValue('--vscode-debugIcon-startForeground'),
+                },
+            },
+            typography: {
+                allVariants: {
+                    fontFamily: computedStyle.getPropertyValue('--vscode-font-family'),
+                },
+            },
+        });
+    };
+
+    const [theme, setTheme] = React.useState<Theme>(createVscodeTheme('light'));
+
+    const respondToMessage = function (message: MessageEvent<VSCodeMessage>) {
+        if (message.data.action === 'setTheme') {
+            setTheme(createVscodeTheme(message.data.themeValue === 1 ? 'light' : 'dark'));
+        } else if (message.data.action === 'setAvailableServices') {
+            setAvailableServices(message.data.availableServices);
+        } else if (message.data.action === 'setComponentName') {
+            setComponentName(message.data.componentName);
+        }
+    };
+
+    React.useEffect(() => {
+        window.addEventListener('message', respondToMessage);
+        return () => {
+            window.removeEventListener('message', respondToMessage);
+        };
+    }, []);
+
+    const handleSubmit = (event: React.FormEvent) => {
+        event.preventDefault();
+        window.vscodeApi.postMessage({
+            action: 'addServiceBinding',
+            params: {
+                selectedService,
+                bindingName,
+            },
+        } as ActionMessage);
+    };
+
+    const isBindingNameValid = (name: string): boolean => {
+        return /^[a-z0-9]([-a-z0-9]*[a-z0-9])?$/.test(name);
+    };
+
+    return (
+        <ThemeProvider theme={theme}>
+            <Container maxWidth="md">
+                {availableServices && componentName ? (
+                    <form onSubmit={handleSubmit}>
+                        <Stack spacing={3} marginTop={3}>
+                            <FormLabel>
+                                Bind Service to <code>{componentName}</code>
```

Comment:
Instead of `componentName` it showed the **component folder name**
![image](https://github.com/redhat-developer/vscode-openshift-tools/assets/93245779/be18ce5e-7714-498a-9038-02bddc03ad8d)